frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -124,31 +138,29 @@ function getParameterPanelBorderColor(theme?: DisplayTheme) {
 export const ParametersWidgetContainer = styled(FullWidthContainer)<{
   embedFrameTheme?: DisplayTheme;
   hasScroll: boolean;
+  canSticky: boolean;
   isSticky: boolean;
 }>`
   padding-top: ${space(1)};
   padding-bottom: ${space(1)};
 
   ${props =>
-    props.hasScroll &&
-    css`
-      border-bottom: 1px solid
-        ${getParameterPanelBorderColor(props.embedFrameTheme)};
-    `}
-
-  ${props =>
-    props.isSticky &&
+    props.canSticky &&
     css`
       position: sticky;
       top: 0;
       left: 0;
       width: 100%;
       z-index: 3;
 
-      background-color: var(
-        --mb-color-embed-bg-color-override,
-        ${getParameterPanelBackgroundColor(props.embedFrameTheme)}
-      );
+      transition: background-color 0.4s;
```

Comment:
For transitioning transparent filters background to semi-transparent filters background in case `background: false`

For example
![chrome_laptop_embed_PublicOrEmbeddedDashboardView_Dark_Theme_No_Background_Scroll](https://github.com/metabase/metabase/assets/1937582/8e25d56b-df17-4662-9876-2924c5756ecd)